SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -435,6 +435,9 @@ void USpatialConnectionManager::FinishConnecting(Worker_ConnectionFuture* Connec
 {
 	TWeakObjectPtr<USpatialConnectionManager> WeakSpatialConnectionManager(this);
 
+	// A pending connection is used to avoid creating objects in the task graph as it can be called during GC (via UMaterialInstace::Destroy
+	// -> FlushRenderingCommands) and trigger an error.
+	PendingWorkerConnection = NewObject<USpatialWorkerConnection>(this);
```

Comment:
I think with `GetWorkerConnection` being a public accessor we need to be careful here and introduce a new temporary